templates/mod/_require.epp

```diff
@@ -0,0 +1,33 @@
+<% $_requires = if $requires { %>$requires<% }else {%>$requires_defaults<%} %>
```

Comment:
can we make sure consistent syntax across the temples?

Reply:
Sure